File: app/src/main/java/com/iomt/android/CharCell.java
package com.iomt.android;

public class CharCell extends AbsCell {
    private String name;
    private String data;

    public CharCell(String name) {
        this.name = name;
        data = "--";
    }

    public String getName() {
        return name;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void reset() {
        data = "--";
    }
}

File: app/src/main/java/com/iomt/android/CharCell.kt
package com.iomt.android

/**
 * @property name name of characteristic cell
 */
data class CharCell(var name: String) : AbstractCell() {
    /**
     * Some data
     */
    var data = "--"

    /**
     * Reset cell data
     */
    fun reset() {
        data = "--"
    }
}
